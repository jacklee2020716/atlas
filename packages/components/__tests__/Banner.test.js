import React from "react"
import { mount } from "enzyme"
import Banner from "./../src/components/Banner"

describe("Banner component", () => {

  const component = mount(<Banner src="http://static.libsyn.com/p/assets/2/c/2/5/2c25acab892a768e/Twitter_Cover.png" />)

  it("Should render correctly", () => {
    expect(component).toMatchSnapshot()
  })

})