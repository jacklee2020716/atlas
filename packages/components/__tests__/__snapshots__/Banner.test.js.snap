// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner component Should render correctly 1`] = `
<Memo(Banner)
  src="http://static.libsyn.com/p/assets/2/c/2/5/2c25acab892a768e/Twitter_Cover.png"
>
  <div
    css={
      Object {
        "map": undefined,
        "name": "p5v1zq",
        "next": undefined,
        "styles": "
    background-image: url(http://static.libsyn.com/p/assets/2/c/2/5/2c25acab892a768e/Twitter_Cover.png);
    background-size: cover;
    background-position: center;
    width: 100%;
    min-height: 6.25rem;
  ",
        "toString": [Function],
      }
    }
  />
</Memo(Banner)>
`;
